### PR TITLE
Add ".postcss" to PostCSS language

### PR DESCRIPTION
**Pull request by Akryum**
## Checklist:

- [x] **I am associating a language with a new file extension.**
  - [x] The new extension is used in hundreds of repositories on GitHub.com
    - Search results for each extension:
      
      - https://github.com/search?q=extension%3Apostcss+NOT+nothack&type=Code
  - [x] I have included a real-world usage sample for all extensions added in this PR:
    - Sample source(s):
      - https://github.com/Akryum/awesomejs.dev/blob/e59a65f6d318cfad9f340681e843d2de20d032fb/packages/frontend/src/assets/styles/components/markdown.postcss#L1
    - Sample license(s):
      - MIT
  - [ ] I have included a change to the heuristics to distinguish my language from others using the same extension.

### CODE REVIEW

File: samples/PostCSS/sample.postcss [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+$big: 100px;
+
+/* Main block */
+.block {
+  &_logo {
+    background: inline("./logo.png");
+    @mixin size $big;
+  }
+}
```
Comment (pchaigno) [inline samples/PostCSS/sample.postcss]: Where is this sample from? It looks very simple. Is it something you wrote specifically for this pull request?

We generally prefer more realistic samples as they are used to train the naive Bayes classifier.

Comment (Akryum) [reply samples/PostCSS/sample.postcss]: It's the existing postcss sample with the new extension. 😸

Comment (pchaigno) [reply samples/PostCSS/sample.postcss]: Ah. Any chance you could find another sample from a "real" application under a permissive license?

Comment (Akryum) [reply samples/PostCSS/sample.postcss]: https://github.com/Akryum/awesomejs.dev/blob/e59a65f6d318cfad9f340681e843d2de20d032fb/packages/frontend/src/assets/styles/components/markdown.postcss#L1 (MIT)

Comment (pchaigno) [reply samples/PostCSS/sample.postcss]: That looks fine :+1:

**Pull request merged**